File: geocoding/src/main/java/se/arbitur/geocoding/Models/Bounds.java
package se.arbitur.geocoding.Models;


public final class Bounds {
	private Coordinate southWest;
	private Coordinate northEast;

	public Bounds(Coordinate southWest, Coordinate northEast) {
		this.southWest = southWest;
		this.northEast = northEast;
	}

	public Bounds(double swLat, double swLong, double neLat, double neLong) {
		southWest = new Coordinate(swLat, swLong);
		northEast = new Coordinate(neLat, neLong);
	}

	@Override
	public String toString() {
		return southWest.toString() + "|" + northEast.toString();
	}
}

File: geocoding/src/main/java/se/arbitur/geocoding/Models/Bounds.kt
package se.arbitur.geocoding.Models


data class Bounds(val southWest: Coordinate, val northEast: Coordinate) {

	constructor(swLat: Double, swLong: Double, neLat: Double, neLong: Double):
			this(Coordinate(swLat, swLong), Coordinate(neLat, neLong))

	override fun toString(): String = "$southWest|$northEast"
}